lib/fluent/plugin/out_google_cloud.rb

```diff
@@ -481,6 +487,7 @@ def initialize
       @retried_entries_count = nil
 
       @ok_code = nil
+      @uptime_time = Time.now.to_i
```

Comment:
Unfortunately, this variable name seems misleading (or at least confusing) now, as the name suggests that it stores the time the agent has been up. Something like `@last_uptime_update_time` would, to me, more accurately describe the implementation.